spec/controllers/assignment_invitations_controller_spec.rb

```diff
@@ -34,7 +34,7 @@
                         title: 'ruby-project',
                         starter_code_repo_id: '1062897',
                         organization: organization,
-                        public_repo: false)
+                        public_repo: true)
```

Comment:
Sadly I haven't yet received the discount pack I applied from GitHub education for me to create a private repo in an organization, so I changed these into public repo for recording VCRs.

I hope this does not cause any inconvenience and will change them back when I get the discount pack.